plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -250,6 +250,20 @@ private function get_default_product_templates() {
 				),
 			)
 		);
+		$templates[] = new ProductTemplate(
+			array(
+				'id'                    => 'variable-product-template',
+				'title'                 => __( 'Variable product', 'woocommerce' ),
+				'description'           => '',
+				'is_selectable_by_user' => false,
+				'order'                 => 15,
+				'icon'                  => 'shipping',
+				'layout_template_id'    => 'simple-product',
+				'product_data'          => array(
+					'type' => 'variable',
+				),
+			)
+		);
```

Comment:
Hmm I am not sure if this is actually what we want to do.
Ideally if we want to support this programmatically, the standard product template should support both `simple` and `variable`.
Maybe type can be an array or a `RegExp`🙄😬